ci-operator/jobs/openshift/ansible-service-broker/openshift-ansible-service-broker-master-presubmits.yaml

```diff
@@ -192,6 +193,278 @@ presubmits:
             cpu: 10m
       serviceAccountName: ci-operator
     trigger: '(?m)^/test (?:.*? )?operator-lint(?: .*?)?$'
+  - agent: kubernetes
```

Comment:
All these new jobs look suspicious, we got rid of the `...__rhel.yaml` variant files a while ago. How did these end up here?

Reply:
I think that was my fault. I had a local file not in the commit when I generated the jobs. 

I removed it and re-generated and nothing changed though.

Reply:
Yeah the generator never removes any jobs, even if it generated them before - you'll need to remove the jobs manually.

Reply:
I removed it and restarted the job. Thanks for the help!